app/src/main/sentences/es/open.dslf

```diff
@@ -0,0 +1,3 @@
+
+open: medium
+abre|inicia|abrir|inicia|iniciar|corre|lanza? (app|aplicacion? .what.)|(.what. app|programa|aplicacion?);
```

Comment:
There is a problem here: anything would match, since everything except the capturing group is optional. Maybe split this into 2+ sentences to be sure there are no such problems